app/feemarket.go

```diff
@@ -0,0 +1,35 @@
+package app
+
+import (
+	storetypes "cosmossdk.io/store/types"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
+	feemarket "github.com/skip-mev/feemarket/x/feemarket"
+	feemarketkeeper "github.com/skip-mev/feemarket/x/feemarket/keeper"
+	feemarkettypes "github.com/skip-mev/feemarket/x/feemarket/types"
+)
+
+// registerFeeMarketModule registers the feemarket module.
+func (app *AlloraApp) registerFeeMarketModule() {
```

Comment:
It's not mentioned in the `feemarket` documentation but it supports auto-wiring using `depinject`: https://github.com/skip-mev/feemarket/blob/f1f216e15aa6bcc05075a7079496764820a0f545/x/feemarket/module.go#L178

I'd rather register it in the `app.yaml` as I consider it the canonical way to register a module, and only fallback to manual wiring when not possible otherwise (e.g. IBC modules).

Reply:
Nice find! I will consider this!